rllib/algorithms/ppo/tf/ppo_tf_rl_module.py

```diff
@@ -40,6 +40,11 @@ def _forward_exploration(self, batch: NestedDict) -> Dict[str, Any]:
         the policy distribution to be used for computing KL divergence between the old
         policy and the new policy during training.
         """
+        # TODO (sven): Make this the only bahevior once PPO has been migrated
+        #  to new API stack (including EnvRunners!).
+        if self.config.model_config_dict.get("uses_new_env_runners"):
```

Comment:
temporary hack to make sure RLModule knows, when it still has to compute vf-preds via `forward_exploration` (old and hybrid API stacks).